### ISSUE TITLE
pnnx结果是对的，转到ncnn后结果出问题了

### ISSUE THREAD

**Issue by XdpAreKid** [Tencent/ncnn#4961]
## error log | 日志或报错信息 | ログ
无
## how to reproduce | 复现步骤 | 再現方法
[模型](https://drive.google.com/file/d/1Ci7kjWw6-x0lKWpCp1X39MYQkjyggbzr/view)
1. 复现的pnnx代码
`
def test_inference():
    net = Model()
    net.eval()

    img = cv2.imread("/home/xdp/code/PaddleOCR2Pytorch/testimg/2.png")
    resized_image = cv2.resize(img, (320, 48))
    resized_image = resized_image.astype('float32')
    resized_image = resized_image.transpose((2, 0, 1)) / 255
    resized_image -= 0.5
    resized_image /= 0.5
    resized_image = resized_image[np.newaxis, :]
    tensor = torch.from_numpy(resized_image)
    torch.manual_seed(0)
    out = net(tensor).detach().numpy()

    preds_idx = out.argmax(axis=2)
    preds_prob = out.max(axis=2)
`
2.复现的ncnn代码
def test_inference():
    out = []
    img = cv2.imread("/home/xdp/code/PaddleOCR2Pytorch/testimg/2.png")
    resized_image = cv2.resize(img, (320, 48))
    resized_image = resized_image.astype('float32')
    resized_image = resized_image.transpose((2, 0, 1)) / 255
    resized_image -= 0.5
    resized_image /= 0.5
    resized_image = resized_image[np.newaxis, :]

    with ncnn.Net() as net:
         net.load_param("./ch_recModel.ncnn.param")
         net.load_model("./ch_recModel.ncnn.bin")

         with net.create_extractor() as ex:
            ex.input("in0", ncnn.Mat(resized_image.squeeze(0)).clone())

            _, out0 = ex.extract("out0")
            out.append(np.array(out0))
    rec_list = []
    preds_idx = out[0].argmax(axis=1)
    preds_prob = out[0].max(axis=1)
3. 两者的preds_idx是不一致的

**Comment by XdpAreKid** [comment]
我搞错了。。。。。
